ecommerce-ui/src/components/cart/cart.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+
+export default class Cart extends Component {
+    render() {
+        const siteList = this.props.sites
+            .map( (site, idx) => {
+                return (
+                    <li key={idx}>
+                        <button onClick={this.props.clickHandler}>Delete</button>
```

Comment:
You need to pass in idx to the function. also you need to prevent it from throwing an error by passing in an anonymous function. you can do this by instead using: 
onClick={ () => this.props.clickHandler(idx) }